javers-spring/src/main/java/org/javers/spring/aspect/JaversAdvice.java

```diff
@@ -0,0 +1,41 @@
+package org.javers.spring.aspect;
+
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+import org.javers.core.Javers;
+
+/**
+ * @author Pawel Szymczyk
+ */
+public class JaversAdvice implements MethodInterceptor {
+
+    private final Javers javers;
+    private final String author;
+
+    public JaversAdvice(Javers javers, String author) {
+        this.javers = javers;
+        this.author = author;
+    }
+
+    @Override
+    public Object invoke(MethodInvocation invocation) throws Throwable {
+
+        Object retVal = invocation.proceed();
+
+        for (Object arg: invocation.getArguments()) {
+            if (arg instanceof Iterable) {
```

Comment:
Akurat w tym miejscu jest sprawdzane czy jeden z argumentów metody jest Iterable, jeżeli jest to encje są wypakowywane z kontenera. Zastanawiałem się nad tym czy dodawać ten feature ale wydaje mi się, że częstą praktyką jest dodawanie w repo metod:

save(E... entities) 

i to jest właśnie obsłużenie tego przypadku.

Natomiast to, że użytkownik może nie chcieć commitować wszystkich arg odnosi się do lini:
```
  for (Object arg: invocation.getArguments()) {
```

i tu rzeczywiście warto będzie pomyśleć o przekazywaniu wartości do adnotacji e.g:

@JaversAuditable(excludeFields={"flag1", "flag2"})

ale, to może w kolejnym PR